examples/js/loaders/ColladaLoader.js

```diff
@@ -19,7 +19,7 @@ THREE.ColladaLoader.prototype = {
 
 		var scope = this;
 
-		var path = scope.path === undefined ? THREE.LoaderUtils.extractUrlBase( url ) : scope.path;
+		var path = ( scope.path === undefined ) ? THREE.LoaderUtils.extractUrlBase( url ) : scope.path;
 
 		var loader = new THREE.FileLoader( scope.manager );
 		loader.load( url, function ( text ) {
```

Comment:
Um, wait. `setPath()` should affect the original file, right? Then we need something like this, correct?
```js
var fileUrl = ( scope.path === undefined ) ? url : ( scope.path + url );

loader.load( fileUrl, function ( text ) {
```